profile/Makefile

```diff
@@ -105,7 +105,7 @@ endif
 
 # MPI wrapped compiler
 ifeq ($(COMPILER),mpi)
-  CC = mpicc
+  CC = mpic++
```

Comment:
Any specific difference? it seems gcc/mpicc works.

Reply:
It works for me too on most machines, using mpich for mpi. But on my local computer, where I installed openmpi instead, mpicc doesn't link the openmpi c++ bindings. mpic++ does.